docs/api.md

```diff
@@ -0,0 +1,531 @@
+- then sends variable updates to the IoT Cloud (via MQTT client)
+- finally, it checks for new data from the IoT Cloud (via the MQTT client)
+
+Any IoT Cloud sketch needs to continuously call the `update()` function, as it will otherwise time out and reset. The `update()` is by default placed inside of the `loop()`.
+
+**Note:** do not use the `delay()` function in the sketch, as the watchdog timer will cause the board to reset and attempt to reconnect.
+
+#### Syntax
+
+```
+ArduinoCloud.update()
+```
+
+#### Parameters
+None.
+
+#### Returns
+Nothing.
+
+
+### `connected()` (TCP)
+
+#### Description
+
+Checks the connection status and returns an integer the connection status (int).
+
+#### Syntax
+
+```
+ArduinoCloud.connected()
+```
+
+#### Parameters
+None.
+
+#### Returns
+- `int` - status of connection
+
+### `printDebugInfo()` (TCP)
+
+#### Description
+
+Print any available debug information.
+
+#### Parameters
+None
+
+#### Returns
+Nothing.
+
+### `setBoardId()`
+
+#### Description
+
+This method is only enabled if you are using an ESP32/ESP8266 board, and sets the board/device id.
+
+#### Syntax
+
+```
+ArduinoCloud.setBoardId(DEVICE_LOGIN_NAME);
+```
+
+### `setSecretDeviceKey()`
+
+#### Description
+
+This method is only enabled if you are using an ESP32/ESP8266 based board, and sets the secret device key for the device. The secret key is only obtainable from the Arduino IoT Cloud during device configuration.
+
+#### Syntax
+
+```
+ArduinoCloud.setSecretDeviceKey(DEVICE_KEY);
+```
+
+### `getBrokerAddress()`
+
+#### Description
+
+Get the MQTT broker address used.
+
+#### Syntax
+
+```
+ArduinoCloud.getBrokerAddress()
+```
+
+#### Parameters
+None.
+
+#### Return
+- `String` - the MQTT broker address.
+
+### `getBrokerPort()`
+
+#### Description
+
+Get the MQTT broker port used.
+
+#### Syntax
+
+```
+ArduinoCloud.getBrokerPort()
+```
+
+#### Parameters
+None.
+
+#### Return
+- `int` - the MQTT broker port.
+
+<!-- LPWAN documentation begins here -->
+
+## ArduinoCloud Class (LPWAN)
+
+### `begin()` (LPWAN)
+
+### Description
+This function initializes the ArduinoIoTCloudLPWAN library with the specified connection handler and retry settings.
+
+#### Parameters
+- `ConnectionHandler` - object containing `APPEUI`, `APPKEY` and frequency band (e.g. `_lora_band::EU868`)
+- `bool` - a boolean value that decides whether retry functionality is enabled (`true`) or disabled (`false`)
+
+#### Returns
+
+- `int` - returns `1` on successful initialization.
+
+### `update()` (LPWAN)
+
+#### Description
+
+**Note:** for TCP devices, the `update()` function is implemented differently. See [ArduinoCloud Class (TCP)](). 
+
+Responsible for updating the IoT Cloud connection. 
+- First it feeds a watchdog timer, making sure it is not stuck anywhere,
+- then sends variable updates to the IoT Cloud (via MQTT client)
+- finally, it checks for new data from the IoT Cloud (via the MQTT client)
```

Comment:
LoRa do not use MQTT protocol

Reply:
Ahh yes this was incorrectly added from the TCP method. Corrected!